spec/lib/reporting/identity_verification_report_spec.rb

```diff
@@ -39,7 +39,7 @@
         { 'user_id' => 'user3', 'name' => 'Fraud: Profile review passed', 'success' => '1' },
 
         # Success through address confirmation user
-        { 'user_id' => 'user4', 'name' => 'IdV: GPO verification submitted' },
+        { 'user_id' => 'user3', 'name' => 'IdV: GPO verification submitted' },
```

Comment:
there's no `user4` anymore, is there a way we could keep user4 but add a duplicate event to user 3 so it still shows what we want the test to show?